### PR TITLE
[irods/irods#3727] Remove ASSERT_* from kerberos auth plugin (main)

### PR DESCRIPTION
**Pull request by alanking**
Tests need to be run

### CODE REVIEW

**Comment by alanking** [comment]
companion PR to irods/irods#6155

**Comment by alanking** [comment]
This builds. Haven't run tests yet, but I don't know that tests have ever been run on main since branching in this repo

File: irods_consortium_continuous_integration_build_hook.py [hunk 1]
```diff
@@ -65,21 +68,25 @@ def copy_output_packages(build_directory, output_root_directory):
         irods_python_ci_utilities.append_os_specific_directory(output_root_directory),
         lambda s:s.endswith(irods_python_ci_utilities.get_package_suffix()))
 
-def main(output_root_directory, irods_packages_root_directory, externals_directory):
+def main(build_directory, output_root_directory, irods_packages_root_directory, externals_directory):
     install_building_dependencies(externals_directory)
     if irods_packages_root_directory:
         irods_python_ci_utilities.install_irods_dev_and_runtime_packages(irods_packages_root_directory)
-    build_directory = tempfile.mkdtemp(prefix='irods_kerberos_plugin_build_directory')
+    build_directory = os.path.abspath(build_directory or tempfile.mkdtemp(prefix='irods_s3_plugin_build_directory'))
```
Comment (alanking) [inline irods_consortium_continuous_integration_build_hook.py:75]: irods_s3_plugin_build_directory -> irods_kerberos_plugin_build_directory

Comment (alanking) [reply irods_consortium_continuous_integration_build_hook.py]: Done

**Review by alanking — COMMENTED** [review]
Okay, I've changed one function to return a different error object rather than simply returning SUCCESS

**Comment by alanking** [comment]
I will add #'s and merge later today unless there are any objections

**Comment by korydraughn** [comment]
I think you're good to go.

**Pull request merged**